tests/conftest.py

```diff
@@ -58,6 +69,16 @@ def tmp_symlink_dir(tmp_path, tmp_sub_dir):
     symlink_dir.unlink()
 
 
+@pytest.fixture
+def chdir_tmp_path(tmp_path):
+    "Make the test run in ``tmp_path`` as the current working directory."
+
+    cur_dir = Path.cwd()
+    os.chdir(tmp_path)
+    yield
+    os.chdir(cur_dir)
```

Comment:
So anything after a `yield` or `return` statement in a fixture gets run after the test is completed?

Reply:
Only after `yield` not `return` https://docs.pytest.org/en/6.1.2/yieldfixture.html

Reply:
thanks